wiki/Changelog/de.md

```diff
@@ -2,29 +2,41 @@
 tags:
   - history
   - Geschichte
-outdated_translation: true
-outdated_since: 5a98d627b0d81c69ab200df06feb80432ce7b24b
 ---
 
 # Änderungsprotokoll
 
-Das **[Änderungsprotokoll](https://osu.ppy.sh/home/changelog)** ist eine Seite, auf der das [osu!-Entwicklerteam](/wiki/People/Developers) ihre täglichen Verbesserungen, Ergänzungen und Fehlerbehebungen des Spiels notiert. Alle Änderungen von früheren Versionen werden hier aus Archivierungszwecken aufgezeichnet.
+Das **[Änderungsprotokoll](https://osu.ppy.sh/home/changelog)** ist eine Seite, auf der das [osu!-Entwicklerteam](/wiki/People/Developers) seine täglichen Verbesserungen, Ergänzungen und Fehlerbehebungen des Spiels sowie der Webseite notiert. Alle Änderungen von früheren Versionen werden hier zu Archivierungszwecken aufgezeichnet.
 
 Die Protokolle für die Webseite und [osu!(lazer)](/wiki/Client/Release_stream/Lazer) werden automatisch basierend auf GitHub-Versionen dieser Projekte generiert, während der Rest manuell gepflegt wird.
 
+## Inhalte
+
+Die Hauptseite des Änderungsprotokolls zeigt Updates von allen osu!-Komponenten in absteigender chronologischer Reihenfolge an. Klicke auf eine der Kategorien, um die Auflistung einzugrenzen (Updates des osu!-Wikis sind in `Web` enthalten). Unter der Auswahlliste befindet sich ein Graph, der die Verteilung der Online-Nutzer für jede [Updatequelle](/wiki/Client/Release_stream) des Clients darstellt. Innerhalb einer Kategorie werden Änderungen nach Themengebieten gruppiert und Änderungen, die verhältnismäßig wichtig sind, sind goldfarben.
+
+Die Änderungsliste unterstützt neben Markdown-Formatierung auch Medien wie statische Bilder, animierte GIFs und eingebettete Videos. Während die Änderungen manuell hinzugefügt werden können, werden sie typischerweise von GitHub (wo eine Überprüfung der Änderungen stattfindet) abgerufen und automatisch gruppiert, sobald eine neue Version veröffentlicht wird. Standardmäßig wird alles unter einer horizontalen Linie (`---`) in der Beschreibung eines Änderungsvorschlags (Pull Request) als detaillierte Erklärung für die Änderung verwendet.
+
+Änderungen am Protokoll selbst können [im GitHub-Repository `ppy/osu-web`](https://github.com/ppy/osu-web/pulls?q=is%3Apr+sort%3Aupdated-desc+label%3Aarea%3Achangelog) angesehen werden.
+
 ## Geschichte
 
+::: Infobox
 ![](img/changelog-web-2007.png "Die Änderungsliste auf der Webseite im Jahr 2007")
+:::
 
 Das Änderungsprotokoll wurde von peppy am 11. September 2007 in einem dafür gedachten Forum-Thread mit dem Titel "[Official Development Changelog](https://osu.ppy.sh/community/forums/topics/15)" gestartet, in dem er wesentliche Änderungen und Fehlerbehebungen auflisten und manchmal Einblicke in anstehende Pläne geben würde.
```

Comment:
missing link that was added to EN, and potentially minor grammar change (not sure if it applies to this translation)